src/Pages/logIn.cshtml.cs

```diff
@@ -1,7 +1,9 @@
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Threading.Tasks;
```

Comment:
Using a few unnecessary dependancies again, and also I believe that LogIn should start with an upper case letter to match our standard :)